.github/workflows/release.yml

```diff
@@ -2,21 +2,32 @@ name: Release npm package
 
 on:
   push:
-    branches:
-      - main
+    branches: [main, dev]
```

Comment:
We don't have a "dev" branch, we have develop.

Also, of this were to get merged as/is, we would also be running a release on pull requests and develop branch. 

Can you move this into its own gtihub actions workflow? That way you can target the build and unit test separately from the release?